src/math/Camera.h

```diff
@@ -5,6 +5,9 @@
 #include "Vec3.h"
 
 #include <cmath>
+#ifndef M_PI
```

Comment:
I'd prefer not duplicating this in both locations; but I didn't appreciate `M_PI` was not standard! Perhaps a `util/Constants.h` that defines this?

Reply:
The problem is that if my current definition of M_PI was moved to util/Constants.h and util/Constants.h was included before cmath, M_PI might get defined twice which isn't allowed unless the two definitions are identical.